python/GafferSceneTest/TweakPlugTest.py

```diff
@@ -300,5 +300,119 @@ def testCreateOutputCounterpart( self ) :
+			( imath.Color3f( 1.0, 2.0, 3.0 ), GafferScene.TweakPlug.Mode.Add, imath.Color3f( 5.0, 7.0, 9.0 ) ),
+			( imath.Color3f( 1.0, 2.0, 3.0 ), GafferScene.TweakPlug.Mode.Subtract, imath.Color3f( 3.0, 3.0, 3.0 ) ),
+			( imath.Color3f( 1.0, 2.0, 3.0 ), GafferScene.TweakPlug.Mode.Multiply, imath.Color3f( 4.0, 10.0, 18.0 ) ),
+			( imath.Color3f( 1.0, 8.0, 3.0 ), GafferScene.TweakPlug.Mode.Min, imath.Color3f( 1.0, 5.0, 3.0 ) ),
+			( imath.Color3f( 1.0, 8.0, 3.0 ), GafferScene.TweakPlug.Mode.Max, imath.Color3f( 4.0, 8.0, 6.0 ) ),
+		] :
+			tweaks[1]["mode"].setValue( mode )
+			tweaks[1]["value"].setValue( value )
+
+			parameters = originalParameters.copy()
+
+			self.assertTrue( tweaks.applyTweaks( parameters ) )
+			self.assertEqual( parameters["b"].value, expectedValue )
+
+		# Arithmetic on a V2f
+
+		for value, mode, expectedValue in [
+			( imath.V2f( 1.0, 2.0 ), GafferScene.TweakPlug.Mode.Replace, imath.V2f( 1.0, 2.0 ) ),
+			( imath.V2f( 1.0, 2.0 ), GafferScene.TweakPlug.Mode.Add, imath.V2f( 4.0, 6.0 ) ),
+			( imath.V2f( 1.0, 2.0 ), GafferScene.TweakPlug.Mode.Subtract, imath.V2f( 2.0, 2.0 ) ),
+			( imath.V2f( 1.0, 2.0 ), GafferScene.TweakPlug.Mode.Multiply, imath.V2f( 3.0, 8.0 ) ),
+			( imath.V2f( 1.0, 5.0 ), GafferScene.TweakPlug.Mode.Min, imath.V2f( 1.0, 4.0 ) ),
+			( imath.V2f( 1.0, 5.0 ), GafferScene.TweakPlug.Mode.Max, imath.V2f( 3.0, 5.0 ) ),
+		] :
+			tweaks[2]["mode"].setValue( mode )
+			tweaks[2]["value"].setValue( value )
+
+			parameters = originalParameters.copy()
+
+			self.assertTrue( tweaks.applyTweaks( parameters ) )
+			self.assertEqual( parameters["c"].value, expectedValue )
+
+		# List operations
+
+		for value, mode, expectedValue in [
+			( [ "brie" ], GafferScene.TweakPlug.Mode.ListAppend, [ "gouda", "cheddar", "swiss", "brie" ] ),
+			( [ "brie", "cheddar", "muenster" ], GafferScene.TweakPlug.Mode.ListAppend, [ "gouda", "swiss", "brie", "cheddar", "muenster" ] ),
+			( [ "cheddar" ], GafferScene.TweakPlug.Mode.ListAppend, [ "gouda", "swiss", "cheddar" ] ),
+			( [ "edam" ], GafferScene.TweakPlug.Mode.ListPrepend, [ "edam", "gouda", "cheddar", "swiss" ] ),
+			( [ "edam", "cheddar", "muenster" ], GafferScene.TweakPlug.Mode.ListPrepend, [ "edam", "cheddar", "muenster", "gouda", "swiss" ] ),
+			( [ "swiss" ], GafferScene.TweakPlug.Mode.ListPrepend, [ "swiss", "gouda", "cheddar" ] ),
+			( [ "gouda" ], GafferScene.TweakPlug.Mode.ListRemove, [ "cheddar", "swiss" ] ),
+			( [ "gouda", "swiss" ], GafferScene.TweakPlug.Mode.ListRemove, [ "cheddar" ] ),
+		] :
+			tweaks[3]["mode"].setValue( mode )
+			tweaks[3]["value"].setValue( IECore.StringVectorData( value ) )
+
+			parameters = originalParameters.copy()
+
+			self.assertTrue( tweaks.applyTweaks( parameters ) )
+			self.assertEqual( parameters["d"], IECore.StringVectorData( expectedValue ) )
+
+		# \todo Handle numeric lists, like float vector, etc. Only for lists, not for adding and such?
```

Comment:
I'd assume that numeric list operations would already work given your current implementation. I think we can ignore arithmetic operations on lists for now, so I think we can remove this todo.

Reply:
Fixed in 28eadea4221e3faf6fe4d8ffad2b0591d78fc7fe